aopy/preproc.py

```diff
@@ -341,7 +341,8 @@ def calc_reward_rate(event_log, event_name='REWARD'):
 
 def trial_separate(events, times, evt_start, n_events=8):
     '''
-    Compute the 2D matrices contaning events per trial and timestamps per trial
+    Compute the 2D matrices contaning events per trial and timestamps per trial. 
+    If there are not enough events to fill n_events, the remaining indices will be missing values.
```

Comment:
Remaining indices will be missing values if the events are strings, -1 if the events are a signed int, and the maximum possible value for an unsigned int. Alternatively, within the function we should cast any unsigned int to a signed int since I can't think of a benefit to inputting an unsigned int.